tests/test_isrFunctions.py

```diff
@@ -104,6 +155,38 @@ def test_makeThresholdMask(self):
 
         self.assertEqual(len(defectList), 1)
 
+    def test_ITLEdgeBleedMask(self):
+        """Expect number of masked pixels according to edge bleed masking.
+        """
+        ITLdetector = TestITLDetector()
```

Comment:
Please don't capitalize object instances.  This needs to be `itlDetector` or even just `detector`.

Reply:
Similarly for the exposure below.